tools/build/setup_macos_android.sh

```diff
@@ -56,12 +61,13 @@ function install_gradle() {
   if which -s gradle; then
     echo 'Gradle already installed'
   else
-    if ! which -s brew; then
-      # TODO(fortuna): Install gradle without Homebrew: https://gradle.org/install/#manually
-      echo 'Homebrew is not installed. You need to manually install it: https://docs.brew.sh/Installation' > 2
-      exit 1
-    fi
-    brew install gradle
+    declare -r gradle_home=${1?Need to pass Gradle home}
```

Comment:
I recommend processing arguments as the first thing in a function, so it's clear.